### PR TITLE
Adds a Collection of Spent Ballistic Casings With Random Rotation and Offsets

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Adds a selection of casings for use in mapping:
* Buckshot, slugs, rubbershot, beanbag, dragonsbreath, stun.
* small, large, and .50 cal bullet casings.
* Spent IK casings.

The USSP station ruin has been remapped with these new spent casings, replacing the varedits it was previously using.

Trash spawners can spawn any of the new casings.

## Why It's Good For The Game
Allows mappers to place down randomly scattered casings without needing to varedit. The random offset and rotation makes them actually look like they've been shot. Good for environmental storytelling.

## Images of changes
image

## Testing
Spawned into USSP ruin, saw the cartridges were now randomly scattered.
Mapped in a pile of all the casing types, they all got scattered on their tiles randomly.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
tweak: mapped-in spent casings are now randomly scattered.
add: Trash spawners can spawn a wider range of spent casings if they roll to spawn a spent ammo casing.
/:cl:

### CODE REVIEW

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -134,5 +131,61 @@
...
+	name = "spent beanbag shell"
+	icon_state = "bshell"
+
+/obj/item/trash/spentcasing/shotgun/slug
+	name = "spent slug shell"
+	icon_state = "blshell"
+
+/obj/item/trash/spentcasing/shotgun/dragonsbreath
+	name = "spent dragonsbreath shell"
+	desc = "A spent shotgun shell. It smells like cordite, burnt plastic, and a hint of petrolium."
```
Comment (lewcc) [inline code/game/objects/items/trash.dm:167]:
```suggestion
	desc = "A spent shotgun shell. It smells like cordite, burnt plastic, and a hint of petroleum."
```

Comment (Fordoxia) [reply code/game/objects/items/trash.dm]: Done.

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
Could you add all the new casing subtypes to the ammo casings list in the trash spawner table?

**Comment by Fordoxia** [comment]
> Could you add all the new casing subtypes to the ammo casings list in the trash spawner table?

Done.

**Review by warriorstar-orion — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**